tests/playwright/src/specs/extension-installation.spec.ts

```diff
@@ -30,6 +30,11 @@ import { WelcomePage } from '../model/pages/welcome-page';
 import { PodmanDesktopRunner } from '../runner/podman-desktop-runner';
 import type { RunnerTestContext } from '../testContext/runner-test-context';
 
+const DISABLED = 'DISABLED';
```

Comment:
why not create an enum with all these states?

Reply:
Agree, that would be for a more work, like refactoring.